### DISCUSSION TITLE
Evento LEAVE + Aclaraciones

### DISCUSSION THREAD

**Discussion by Fer1016** [IIC2133-PUC/2024-2#56] [Tarea 0] [answered]
Hola! En el evento LEAVE no me queda claro a que se refiere con que "... este pingüino siempre estará presente dentro de la tienda", significa que solo debemos imprimir que salió sin modificar la cantidad de pinguinos, puflles...? O que debemos guardar la info del pinguino y luego hacer estas modificaciones?
Tengo una duda similar con respecto la parte de aclaraciones donde dice que "En caso que un pingüino no pueda entrar dado que la tienda esté llena, igual se entregarán los datos de los puffles pertenecientes a este pingüino". Tengo que guardar sus datos sin modificar la cantidad de puffles y pinguinos?
De antemano muchas gracias :)

**Comment by tomastrivino** [comment, marked as answer]
1.- Ver la discusión #7 
2.- Se refiere que, aunque la tienda esté llena y no puedas agregar al pingüino fulanito, recibirás un input como el siguiente:
``` 
ENTER {ID_penguin} {N}
{ID_puffle} {color} ...
{ID_puffle} {color}
```
Es decir, incluso si detectas que este pingüino no podrá ser admitido (y, en consecuencia tampoco sus puffles), igual te entregarán N líneas con los puffles respectivos (por lo tanto, **deberás leer esas líneas** en tu programa).

Espero que esto te sirva de aclaración,
saludos!
